yarn.lock

```diff
@@ -3439,20 +3439,20 @@
   dependencies:
     node-fetch "*"
 
-"@types/node@*", "@types/node@^14.14.31":
-  version "14.14.41"
-  resolved "https://registry.npmjs.org/@types/node/-/node-14.14.41.tgz"
-  integrity sha512-dueRKfaJL4RTtSa7bWeTK1M+VH+Gns73oCgzvYfHZywRCoPSd8EkXBL0mZ9unPTveBn+D9phZBaxuzpwjWkW0g==
+"@types/node@*", "@types/node@>=13.7.0":
+  version "18.11.11"
+  resolved "https://registry.yarnpkg.com/@types/node/-/node-18.11.11.tgz#1d455ac0211549a8409d3cdb371cd55cc971e8dc"
+  integrity sha512-KJ021B1nlQUBLopzZmPBVuGU9un7WJd/W4ya7Ih02B4Uwky5Nja0yGYav2EfYIk0RR2Q9oVhf60S2XR1BCWJ2g==
 
 "@types/node@11.11.6":
   version "11.11.6"
   resolved "https://registry.npmjs.org/@types/node/-/node-11.11.6.tgz"
   integrity sha512-Exw4yUWMBXM3X+8oqzJNRqZSwUAaS4+7NdvHqQuFi/d+synz++xmX3QIf+BFqneW8N31R8Ky+sikfZUXq07ggQ==
 
-"@types/node@>=13.7.0":
-  version "16.11.4"
-  resolved "https://registry.yarnpkg.com/@types/node/-/node-16.11.4.tgz#90771124822d6663814f7c1c9b45a6654d8fd964"
-  integrity sha512-TMgXmy0v2xWyuCSCJM6NCna2snndD8yvQF67J29ipdzMcsPa9u+o0tjF5+EQNdhcuZplYuouYqpc4zcd5I6amQ==
+"@types/node@^14.14.31":
+  version "14.14.41"
+  resolved "https://registry.npmjs.org/@types/node/-/node-14.14.41.tgz"
```

Comment:
why did we even have two versions of @types/nodejs

Reply:
```sh
$ yarn why @types/node
yarn why v1.22.19
[1/4] Why do we have the module "@types/node"...?
[2/4] Initialising dependency graph...
warning Resolution field "typescript@4.9.4" is incompatible with requested version "typescript@^3.0.0"
[3/4] Finding dependency...
[4/4] Calculating file sizes...

=> Found "@types/node@14.14.41"
info Has been hoisted to "@types/node"
info Reasons this module exists
   - Hoisted from "@jest#console#@types#node"
   - Hoisted from "@jest#types#@types#node"
   - Hoisted from "jest-util#@types#node"
   - Hoisted from "jest-watcher#@types#node"
   - Hoisted from "jest-environment-jsdom#@types#node"
   - Hoisted from "cypress#@types#node"
   - Hoisted from "jest#@jest#core#@types#node"
   - Hoisted from "jest-environment-jsdom#@jest#environment#@types#node"
   - Hoisted from "jest-environment-jsdom#@jest#fake-timers#@types#node"
   - Hoisted from "jest-environment-jsdom#@types#jsdom#@types#node"
   - Hoisted from "jest-environment-jsdom#jest-mock#@types#node"
   - Hoisted from "jest-watch-typeahead#jest-watcher#@types#node"
   - Hoisted from "jest#@jest#core#@jest#reporters#@types#node"
   - Hoisted from "jest-watch-typeahead#jest-watcher#@jest#types#@types#node"
   - Hoisted from "cypress#extract-zip#@types#yauzl#@types#node"
   - Hoisted from "jest#@jest#core#jest-haste-map#@types#node"
   - Hoisted from "jest#@jest#core#jest-runner#@types#node"
   - Hoisted from "jest#@jest#core#jest-runtime#@types#node"
   - Hoisted from "@types#jest#expect#jest-util#@types#node"
   - Hoisted from "jest-watch-typeahead#jest-watcher#jest-util#@types#node"
   - Hoisted from "jest-watch-typeahead#jest-watcher#@jest#test-result#@jest#console#@types#node"
   - Hoisted from "@types#jest-when#@types#jest#pretty-format#@jest#types#@types#node"
   - Hoisted from "@types#testing-library__jest-dom#@types#jest#pretty-format#@jest#types#@types#node"
   - Hoisted from "@testing-library#cypress#@testing-library#dom#pretty-format#@jest#types#@types#node"
   - Hoisted from "@types#jest#expect#jest-message-util#@jest#types#@types#node"
   - Hoisted from "jest#@jest#core#jest-haste-map#@types#graceful-fs#@types#node"
   - Hoisted from "jest#@jest#core#jest-config#jest-circus#@types#node"
   - Hoisted from "jest#@jest#core#jest-config#jest-environment-node#@types#node"
   - Hoisted from "jest#@jest#core#@jest#reporters#jest-worker#@types#node"
   - Hoisted from "@testing-library#jest-dom#@types#testing-library__jest-dom#@types#jest#pretty-format#@jest#types#@types#node"
info Disk size without dependencies: "856KB"
info Disk size with unique dependencies: "856KB"
info Disk size with transitive dependencies: "856KB"
info Number of shared dependencies: 0

=> Found "bip39#@types/node@11.11.6"
info This module exists because "bip39" depends on it.
info Disk size without dependencies: "716KB"
info Disk size with unique dependencies: "716KB"
info Disk size with transitive dependencies: "716KB"
info Number of shared dependencies: 0

=> Found "protobufjs#@types/node@16.11.4"
info This module exists because "@oasisprotocol#client#protobufjs" depends on it.
info Disk size without dependencies: "1.7MB"
info Disk size with unique dependencies: "1.7MB"
info Disk size with transitive dependencies: "1.7MB"
info Number of shared dependencies: 0
Done in 0.73s.
```